src/main/java/co/pragra/attendancesystem/entity/Cohort.java

```diff
@@ -26,8 +26,10 @@ public class Cohort {
     private Date startDate;
     @Column(name = "END_DATE", nullable = false)
     private Date endDate;
-    @ManyToMany
+
+    @ManyToMany(cascade = CascadeType.ALL)
     private List<Student> students;
-    @ManyToMany
+
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
For now, let's make is a simple uni-directional relationship.
Let's keep @ManyToMany with no cascade